src/Landing/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import FacebookButton from "../auth/socialAuth/facebook/components/Facebook";
+import GoogleButton from "../auth/socialAuth/google/components/Google";
+import axios from "axios";
+
+let errorMessage = "";
+
+export default class SocialAuthentication extends Component {
```

Comment:
Where are the tests for this file?

Reply:
@AGMETEOR this is a provisional file that will be deleted, after integration of the buttons in the various login and signup form respectively.